ex02.ex

```diff
@@ -53,8 +55,10 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
-
+  first2equal = fn ([h|t]) ->
```

Comment:
## -2

You could use pattern matching in the original function to simplify this:

~~~ elixir
  first2equal = fn
                  [ a, b | _ ]              -> a == b
  end
~~~